crates/egui/src/containers/window.rs

```diff
@@ -419,7 +423,11 @@ impl<'open> Window<'open> {
         // Calculate roughly how much larger the window size is compared to the inner rect
         let (title_bar_height, title_content_spacing) = if with_title_bar {
             let style = ctx.style();
-            let window_margin = style.spacing.window_margin;
+            let window_margin = if is_frame_overrode {
+                frame_margin
+            } else {
+                style.spacing.window_margin
```

Comment:
It feels like `Frame::window` should just set this as `inner_margin` instead of having an `if` statement here

Reply:
EDIT: `Frame::window` already has this as the window margin. You can just use `frame.inner_margin` here

Reply:
OK, it has been modified, please check again.